src/main/webapp/WEB-INF/view/add-page.jsp

```diff
@@ -0,0 +1,21 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
```

Comment:
You don't need it. Your service have to work as API, and you need to test it through postman for example.